packages/uswds-core/src/js/start.js

```diff
@@ -15,13 +13,16 @@ const svg4everybody = require("./polyfills/svg4everybody");
 
 uswds.components = components;
 
-domready(() => {
+const initComponents = () => {
   const target = document.body;
   Object.keys(components).forEach((key) => {
     const behavior = components[key];
     behavior.on(target);
   });
   svg4everybody();
-});
+};
+
+window.addEventListener("DOMContentLoaded", initComponents, { once: true });
```

Comment:
Not sure how much of a problem it would be, but `domready` also handled a scenario where the page is already loaded by the time the script is run, by checking [`document.readyState`](https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState). Should that be accounted for in this replacement implementation?

It may only be an issue if the script was added after the initial page load, and I'm not sure how common that would be.

Reply:
Thanks for the review, @aduth. It does seems like a rare situation, but it could be worth duplicating the functionality we're replacing.

Reply:
@aduth I thought `DOMContentLoaded` and `document.readyState` were interchangeable, but I understand the concern.

> It may only be an issue if the script was added after the initial page load, and I'm not sure how common that would be.

Like adding the `start.js` or `uswds.js` script dynamically?

Reply:
Yeah, the worry is that if `DOMContentLoaded` is only fired once at initial page load, if the script containing this code was added any time after that (for whatever reason), the component initialization would never be run because the event would never be fired again.

Reply:
Updated based on this [MDN example](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event#checking_whether_loading_is_already_complete)

And created a very simple test in https://github.com/mejiaj/gulp-sandbox/tree/jm-test-uswds-replace-domready. Should work the same now based on your suggestion, thanks for taking a look!